src/frontends/tensorflow/src/utils.cpp

```diff
@@ -62,3 +66,120 @@ ov::op::PadType ov::frontend::tensorflow::convert_conv_tf_padding(const ov::fron
+            pads_begin.push_back(tf_explicit_paddings[4]);
+            pads_begin.push_back(tf_explicit_paddings[6]);
+            pads_end.push_back(tf_explicit_paddings[3]);
+            pads_end.push_back(tf_explicit_paddings[5]);
+            pads_end.push_back(tf_explicit_paddings[7]);
+        } else {
+            // For NCDHW layout, explicit paddings has the following form:
+            // [0, 0, 0, 0, pad_d1, pad_d2, pad_h1, pad_h2, pad_w1, pad_w2]
+            pads_begin.push_back(tf_explicit_paddings[4]);
+            pads_begin.push_back(tf_explicit_paddings[6]);
+            pads_begin.push_back(tf_explicit_paddings[8]);
+            pads_end.push_back(tf_explicit_paddings[5]);
+            pads_end.push_back(tf_explicit_paddings[7]);
+            pads_end.push_back(tf_explicit_paddings[9]);
+        }
+    }
+}
+
+ov::OutputVector ov::frontend::tensorflow::translate_convolution_op(const ov::frontend::tensorflow::NodeContext& node,
+                                                                    size_t spatial_dims_num) {
+    TENSORFLOW_OP_VALIDATION(node,
+                             spatial_dims_num == 2 || spatial_dims_num == 3,
+                             "Conv2D or Conv3D are supported only.");
+    TENSORFLOW_OP_VALIDATION(node, node.get_input_size() >= 2, "Convolution must have at least two inputs.");
+    auto input = node.get_input(0);
+    auto filter = node.get_input(1);
+
+    // retrieve attributes for Conv2D
+    auto tf_strides = node.get_attribute<std::vector<int64_t>>("strides");
+    auto tf_padding_type = node.get_attribute<std::string>("padding");
+    ov::op::PadType auto_pad = convert_conv_tf_padding(node, tf_padding_type);
+
+    // retrieve optional attributes
+    auto tf_data_format = node.get_attribute<std::string>("data_format", spatial_dims_num == 2 ? "NHWC" : "NDHWC");
+    auto tf_explicit_paddings = std::vector<int64_t>{};
+    if (auto_pad == ov::op::PadType::EXPLICIT) {
+        tf_explicit_paddings = node.get_attribute<std::vector<int64_t>>("explicit_paddings", {});
+    }
+    std::vector<int64_t> dilation_2d = {1, 1, 1, 1};
+    std::vector<int64_t> dilation_3d = {1, 1, 1, 1, 1};
+    auto tf_dilations =
+        node.get_attribute<std::vector<int64_t>>("dilations", spatial_dims_num == 2 ? dilation_2d : dilation_3d);
+
+    bool is_nhwc = true;
+    if (spatial_dims_num == 2) {
+        TENSORFLOW_OP_VALIDATION(node,
+                                 tf_data_format == "NHWC" || tf_data_format == "NCHW",
+                                 "Conv2D data format is neither NHWC nor NCHW");
+        is_nhwc = (tf_data_format == "NHWC");
+    } else {
+        TENSORFLOW_OP_VALIDATION(node,
+                                 tf_data_format == "NDHWC" || tf_data_format == "NCDHW",
+                                 "Conv3D data format is neither NDHWC nor NCDHW");
+        is_nhwc = (tf_data_format == "NDHWC");
+    }
+
+    // prepare attributes for OpenVINO ConvolutionBackpropData
```

Comment:
ConvolutionBackpropData?
looks like copy-pasted comment :)

Reply:
nit: Will be fixed in the next PR